include/core/parameters.h

```diff
@@ -1332,5 +1332,18 @@ namespace Parameters
     parse_parameters(ParameterHandler &prm);
   };
 
+  /**
+   * @brief Return the tensor of entry @p entry_string. If the entry was changed,
+   * then the changed value is returned, otherwise the default value is
+   * returned. If the entry is not a Tensor, a Throw_error will occur.
+   * This function can be use for Point variables.
+   *
+   * @param prm A parameter handler which is used to parse the simulation information
+   * @param entry_string A declare string
+   */
+  Tensor<1, 3>
+  entry_string_to_tensor3(ParameterHandler  &prm,
+                          const std::string &entry_string);
```

Comment:
That's an ok design, but why don't you just pass the string?
You could have string_to_tensor3 and pass the const string. You would call it by saying  string_to_tensor3(prm.get_string("parameter")). This would be a lot cleaner no? This way. you could move it to utilities.h and also inline the function.